unpackaged/config/qa/classes/QAHelper.cls

```diff
@@ -0,0 +1,23 @@
+/**
```

Comment:
Another option (at the team's discretion) is to run this as Anonymous Apex so that the class is not deployed into the org. Having the class deployed can sometimes cause confusion.